### PR TITLE
Charts development

### LINKED ISSUE #333: Finalize Temperature Line Chart
# Depends on #23 

## Summary

- [ ] temperature line chart queries gaia storage for all temperatures, **or better yet** stores an array of all temperatures as a new redux state variable after initial database query in HealthLog 
- [ ] temperature line chart looks similar to wireframe spec
- [ ] chart in placed under the toggle component

## Motivation

This is one of the necessary components in the wireframe

## Describe alternatives you've considered

Alternatives for how to get user data is specified above - you should defend your choice in your PR

### LINKED ISSUE #339: Create Bar Chart for Symptoms
# Depends on #23 

## Summary

- [ ] Creates bar chart using package of your choosing (but be prepared to defend why you chose it in PR)
- [ ] **OPTIONAL but preferred** on initial database query in health log, create arrays of temperature, fever, sore throat, etc as new redux state variables
- [ ] Displays this bar chart in Symptom toggle

## Motivation

This is a key component of the wireframe spec

## Describe alternatives you've considered

There's a few ways to best implement this. Be prepared to defend your choice and if you didn't choose to do the OPTIONAL step then why

### PR DESCRIPTION
**Pull request by pavel-ilin**
fixes #339 fixes #333 

I intend to reuse Chart.js component for all types of Charts we need. Chart component render chart type based on the  ```chartType``` props we sending to it.

### Please provide enough information so that others can review your pull request:

- moved dummy data for Chart component from the ```rootReducer``` to the ```temperatureReducer```. 
- wired ```temperatureReducer``` with Chart component
- created ```chartType.js``` in the utils directory to store strings of the ```Chart.js``` types
- modify tests for Chart component to unsure that new reducer working and also to incorporate ```PropTypes``` check. 

Line:
test

Bars:
test

### Test plan (required)

yarn test

### Final Checklist

- [x] Have you bumped the version in `package.json`?
    - Second decimal for major change, third decimal for minor change. This can go past 10 i.e. 1.0.9 !=> 1.1.0, 1.0.9 => 1.0.10
    - [Read here for more info](https://semver.org/)
- [x] Have you added any new tests necessary?
- [x] Is your PR rebased off the most current master?
- [x] Have you squashed all commits, or if you will merge will you squash all commits?
- [x] Did you use yarn, not npm?
- [x] Did you use Material-UI wherever possible?
- [x] Did you format according to Prettier?
- [x] Did you run all of your most recent changes locally to make sure everything is working?

### CODE REVIEW

**Comment by SomeMoosery** [comment]
These changes are looking good! Could you just add some screenshots and bump the `package.json` version when you get a chance?

File: client/package.json [hunk 1]
```diff
@@ -97,4 +97,4 @@
       "last 1 safari version"
     ]
   }
-}
```
Comment (SomeMoosery) [inline client/package.json:100]: I'm not sure if you meant to add this, but when bumping the version we just want to bump [this](https://github.com/COVID-19-electronic-health-system/Corona-tracker/blob/cf3351f2955ebd2981b5d633d9afcf9b7647c5a7/client/package.json#L3) line, 0.2.7->0.2.8 👍

File: client/src/components/Chart.jsx [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
-    type: 'line',
+    type: chartType,
     // passing data to out chart
     data: {
       // labels for X axe, inserting labels data from the store
-      labels: dataSample.labels,
+      labels: temperatureData.labels,
       datasets: [
         {
           // title of the dataset
```
Comment (SomeMoosery) [inline client/src/components/Chart.jsx:20]: I think we can remove these comments as the code is self-explanatory enough 👍

File: client/src/components/Chart.jsx [hunk 3]
```diff
@@ -24,7 +24,7 @@ const Chart = () => {
           // Do not fill space bitween X and Y axes
           fill: false,
           // data for Y axe, inserting values data from the store
```
Comment (SomeMoosery) [inline client/src/components/Chart.jsx:26]: Can remove these comments as well

File: client/package.json (hunk 1 above)
Comment (pavel-ilin) [reply client/package.json:100]: Done

File: client/src/components/Chart.jsx (hunk 2 above)
Comment (pavel-ilin) [reply client/src/components/Chart.jsx]: Done

File: client/src/components/Chart.jsx (hunk 3 above)
Comment (pavel-ilin) [reply client/src/components/Chart.jsx]: done

**Pull request merged**